### ISSUE TITLE
Clarification on number of updates recieved in Nation BeLL

### ISSUE THREAD

**Issue by rhuturajm** [open-learning-exchange/open-learning-exchange.github.io#1750]
### Problem
In vi-nation.md, under Sync With the Nation, it says "In Step 4, you did 3 kinds of updates that you receive on the community side: updates, publications, and surveys." But in Step 4, the vi dont receive any updates on surveys.

### Steps to reproduce the problem

N/A

### Screenshots

<img width="791" alt="untdsfitled" src="https://user-images.githubusercontent.com/38227119/39024182-1cbd4f6e-440d-11e8-8eb7-58d537881045.png">

### Proposed solution

The correct sentence would be: 
"In [Step 4](vi-bellapps.md#Different_Kinds_of_Updates_to_Your_Community), you did 2 kinds of updates that you receive on the community side: updates, publications."

**Comment by xyb994** [comment]
@rhuturajm 

You are right, the description is not accurate. Survey is meant to sent at the 9th step at this time.

Can we change [Nation BeLL - sync with nation](https://open-learning-exchange.github.io/#!pages/vi/vi-nation.md#Sync_With_the_Nation) section it to

> In Step 4, we showed you 3 kinds of updates that you may receive..."

**Comment by rhuturajm** [comment]
All right makes sense !!
"In Step 4, you did 3 kinds of updates that you may receive on the community side: updates, publications, and surveys."

**Comment by xyb994** [comment]
@rhuturajm The thing is that BeLL is on feature freeze so there will not be update to BeLL when interns create their community. That's why I said "we showed you 3 kinds of updates that you may receive". vi-bellapps.md [will also be changed](https://github.com/open-learning-exchange/open-learning-exchange.github.io/pull/1692/commits/4d00caec86bc674b0b46420829110c33c35ce14c) with this in mind.

**Comment by rhuturajm** [comment]
Sure, so shall i go ahead with pull request for vi-nations.md? Also if its required I can put in a pull request for vi-bellapps.md with required modifications.
